buildconfig/Jenkins/Conda/package-conda

```diff
@@ -64,6 +65,7 @@ do
         --build-qt) ENABLE_BUILD_QT=true ;;
         --build-workbench) ENABLE_BUILD_WORKBENCH=true ;;
         --build-docs) ENABLE_BUILD_DOCS=true ;;
+        --build-workbench-without-docs) BUILD_WORKBENCH_WITH_DOCS=false ;;
```

Comment:
Can we just also set `ENABLE_BUILD_WORKBENCH` to true here so that passing the single flag will be enough?